src/ui/UnknownMessageItemBody/index.tsx

```diff
@@ -2,13 +2,14 @@ import React, { ReactElement, useContext } from 'react';
 import './index.scss';
 
 import Label, { LabelTypography, LabelColors } from '../Label';
-import { CoreMessageType, getClassName } from '../../utils';
+import { getClassName } from '../../utils';
 import { LocalizationContext } from '../../lib/LocalizationContext';
+import { FileMessage, UserMessage } from '@sendbird/chat/message';
 
 interface Props {
   className?: string | Array<string>;
   isByMe?: boolean;
-  message: CoreMessageType;
+  message: UserMessage | FileMessage | any;
```

Comment:
can we exclude `any;`

Reply:
Then how about using `unknown`?

Reply:
Let me merge this PR first, we can change in the next PR